test/test_fsdp.py

```diff
@@ -97,12 +99,34 @@ def fsdp_main(rank, world_size, args):
     bsz_local_end = int((rank + 1) / world_size * B)
     ref_input_local = ref_input_global[bsz_local_start:bsz_local_end].to(rank)
 
-    for _ in range(N_ITER):
+    # We first run one iteration without compile, as a workaround to compile float8 layer.
+    # In the first iter, float8 layers go to the branches of "self.is_amax_initialized == False"
+    # After that, float8 layers go the the branches of "self.is_amax_initialized == True"
+    # TODO: Need to fix compile to run wihtout this workaround.
```

Comment:
could you paste the stack trace of the error?

Reply:
I commented the stack trace on the linked diff.